bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/DownloadImpl.java

```diff
@@ -123,9 +123,6 @@ protected void initModel() {
         descriptionFromAsset = properties.get(PN_DESCRIPTION_FROM_ASSET, descriptionFromAsset);
         inline = properties.get(PN_INLINE, inline);
         if (currentStyle != null) {
-            if (StringUtils.isBlank(actionText)) {
-                actionText = currentStyle.get(PN_ACTION_TEXT, String.class);
```

Comment:
Shouldn't you also remove `PN_ACTION_TEXT` constant?

Reply:
This one is still used in the edit dialog

Reply:
Hm, you're right, we should then keep the constant in the interface, in case people want to reference the property in their implementation. So you can rollback 303098b, sorry!